sys-devel/gcc/gcc-5.4.0_2016_07_08.recipe

```diff
@@ -0,0 +1,347 @@
+SUMMARY="C/C++ compiler for target ${effectiveTargetMachineTriple}"
+DESCRIPTION="The standard compiler for non-legacy Haiku (i.e. for all \
+architectures other than x86_gcc2)."
+HOMEPAGE="http://gcc.gnu.org"
+
+SOURCE_URI="https://ftp.gnu.org/gnu/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```

Comment:
You might wish to use this:
```
SOURCE_URI="http://ftpmirror.gnu.org/gcc/gcc-5.4.0/gcc-5.4.0.tar.bz2"
```
or this:
```
SOURCE_URI="http://ftpmirror.gnu.org/gcc/gcc-${portVersion%%_*}/gcc-${portVersion%%_*}.tar.bz2"
```
And for HOMEPAGE you might prefer to use this:
```
HOMEPAGE="https://gcc.gnu.org/"
```